napari/_qt/layer_controls/qt_image_controls.py

```diff
@@ -336,11 +335,7 @@ def _toggle_plane_parameter_visibility(self):
             self.planeThicknessLabel.show()
 
     def _update_interpolation_combo(self):
-        interp_names = (
-            Interpolation3D.keys()
-            if self.layer._ndisplay == 3
-            else [i.value for i in Interpolation.view_subset()]
-        )
+        interp_names = [i.value for i in Interpolation.view_subset()]
```

Comment:
I'm not sure, but You told earlier that some rendering types are slow in 3D. Did I correct? If yes, then did we want to expose it to GUI?

Reply:
As long as the default is something fast, I'm OK with just leaving them all in. For some users 3D performance might be good enough for various reasons (e.g. their data is small or they have a really fast graphics card).

Reply:
The risk of that is that a user might unkowingly pick a *very* slow one and grind napari to a halt :P

Reply:
Though I guess for that we could learn something from #4764 and roll back to something sane if the FPS drop too much...